frameworks/elastic/docs/x-pack.md

```diff
@@ -0,0 +1,25 @@
+---
+post_title: X-Pack
+menu_order: 31
+feature_maturity: preview
+enterprise: 'no'
+---
+
+# Elastic X-Pack
+
+[X-Pack](https://www.elastic.co/guide/en/x-pack/current/xpack-introduction.html) is an Elastic Stack extension that bundles security, alerting, monitoring, reporting, and graph capabilities into one easy-to-install package. X-Pack is a commercial product of Elastic that requires a license. By default, X-Pack is not installed as part of the DC/OS Elastic service. However, it's easy to enable X-Pack as part of the service configuration:
+
+![x-pack](img/x-pack.png)
+
+You can toggle this setting at any time. This gives you the option of launching an Elastic cluster without X-Pack and then later enabling it. Or, you can run a cluster with X-Pack enabled to try out the commercial features and, if at the end of the 30-day trial period you don't wish to purchase a license, you can simply disable it. You will not lose access to your data.
+  
+## License Expiration
+
+If you uninstall the X-Pack plugin via the service configuration or you let your license expire, remember these two important points:
+ 
+1. Your data is still there.
+2. All data operations (read and write) continue to work.
+
+Graph, Machine Learning, Alerting and Notification, Monitoring, and Security all operate with reduced functionality when X-Pack becomes unavailable. Regarding security/authentication, any Elastic users you have created will no longer be relevant, as any credentials passed in will be ignored.
```

Comment:
Are "Graph, Machine Learning" proper nouns? If not, they should be lower case.

What do you mean by users no longer being "relevant"?